compiler/packages/babel-plugin-react-compiler/src/CompilerError.ts

```diff
@@ -505,122 +499,168 @@ function printCodeFrame(
   );
 }
 
-function printErrorSummary(severity: ErrorSeverity, message: string): string {
-  let severityCategory: string;
-  switch (severity) {
-    case ErrorSeverity.InvalidConfig:
-    case ErrorSeverity.InvalidJS:
-    case ErrorSeverity.InvalidReact:
-    case ErrorSeverity.UnsupportedJS: {
-      severityCategory = 'Error';
+function printErrorSummary(category: ErrorCategory, message: string): string {
```

Comment:
maybe we should make this a property of the ErrorCategory details object?

Reply:
Will address in a follow up